changes/pr5008.yaml

```diff
@@ -0,0 +1,2 @@
+ docs:
+  -  Added a suggestion in the tutorial to create and use a virtual environment
```

Comment:
Can you drop the changelog entry? It's not in a valid format and the linter is complaining / we don't usually do entries for docs changes.